packages/components/src/components/contract-card/contract-card-items/contract-card-body.tsx

```diff
@@ -68,13 +67,13 @@ const ContractCardBody = ({
 
     const toggle_card_dialog_props = {
         addToast,
-        connectWithContractUpdate,
         current_focus,
         error_message_alignment,
         getContractById,
         onMouseLeave,
         removeToast,
         setCurrentFocus,
+        totalProfit: is_multiplier && !isNaN(Number(profit)) ? getTotalProfit(contract_info) : Number(profit),
```

Comment:
bug fix: Passing the `totalProfit` as a prop is essential to keep 'Total profit/loss' value always updated in mobile contract update form. This issue is that currently it's getting updated only when a user interacts with the form.